protostar-rust/src/lib.rs

```diff
@@ -18,6 +20,34 @@ use crate::test_stats::TestsStats;
 pub mod pretty_printing;
 mod test_stats;
 
+#[derive(Deserialize, Debug, PartialEq)]
+pub struct RunnerConfig {
+    test_name_filter: Option<String>,
+    exact_match: bool,
+    protostar_test_config: ProtostarTestConfig,
```

Comment:
Why this is a nested structure?

Reply:
`ProtostarTestConfig` is resolved from `Scarb.toml`, other arguments are cli only, because it doesn't make sense to have them in the toml imo.